### PR TITLE
Created Pro4U homepage in landing app

### PR DESCRIPTION
**Pull request by guybeckenstein**
# #79
## Changes and additions
- Renamed `landing/templates/html/` to `landing/templates/landing`.
- Created an updated landing homepage for Pro4U which shows professionals for guests & registered users.
- Designed **HTML** template files using bootstrap and **minimal CSS file** within `static`.
   - Bootstrap Javascript was used to enable dropdown menu (user bar).
- Updated `Pro4U/urls.py`.
- Created a new method within `account.models.professionals.py` called `get_avg_rating` to show average rating for each professional entity in homepage.
- Added TODOs where relevant

## View logic
- A guest/user can see up to 48 randomly-shuffled professionals. Each professional has a full name, city, profession, average rating and image. Showing image is not implemented, so we use a default image 'NO IMAGE AVAILABLE' for each professional without image.
- A guest can sign up/log in. Sign up and log in templates are not implemented.
- A guest can search a professional. Search logic is not implemented.

## Tests
Same test logic for template and view, as `learn_more` view.

### CODE REVIEW

**Review by idosinger98 — COMMENTED** [review]
Good job. Only if you can put a professional in the admin and show us how you display his pictures and how when I click on him it takes me to his home page (even make the pro a simple home page that seems to work)

File: landing/views.py [hunk 1]
```diff
@@ -1,8 +1,35 @@
...
 
-def indexView(request):
-    return render(request, 'html/index.html')
+def round_avg_rating(professionals):
+    """
+    f(3.55555555555555)=3.55 -> by rounding. It rounds the avg rating of a professional.
+    """
+    for professional in professionals:
+        if professional.avg_rating is not None:
+            professional.avg_rating = round(professional.avg_rating, 2)
```
Comment (TalReinfeld) [inline landing/views.py:33]: I think that its more correct to round it with only 1 digit after the point
I believe that it works like that:
professional.avg_rating = round(professional.avg_rating, 1)

**Comment by TalReinfeld** [comment]
It failed in the PyTest...

**Comment by Yarboa** [comment]
> It failed in the PyTest...

Please remove labels in case status moved to Draft,

**Comment by Yarboa** [comment]
> It failed in the PyTest...

Please take a look here 
https://github.com/redhat-beyond/Pro4U/actions/runs/4907225025/jobs/8762214599?pr=83#step:6:371-373

File: landing/tests/tests.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import pytest
+from django.test import Client
```
Comment (Yarboa) [inline landing/tests/tests.py:2]: It is not the correct way to use client fixtures in pytest
Please take a look here
https://pytest-django.readthedocs.io/en/latest/helpers.html#id2

Do not confuse with this 
https://docs.djangoproject.com/en/4.2/topics/testing/tools/

Read the pytest-django docs, it is wrapping the django-test client

**Review by Yarboa — CHANGES_REQUESTED** [review]

File: landing/templates/landing/base.html [hunk 3]
```diff
@@ -0,0 +1,159 @@
+{% load static %}
+
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
```
[the remark this answers is not in the document]
Comment (Yarboa) [reply landing/templates/landing/base.html:5]: you have too many css files,
Once working with boostrap, most of css changes done through bootstrap
less css files please

[the remark this answers is not in the document]
Comment (Yarboa) [reply landing/templates/landing/base.html]: I will not die on that hill
but all of the css staff making the code non readable

**Review by Yarboa — APPROVED** [review]
LGTM

**Pull request merged**